plc4j/api/src/main/java/org/apache/plc4x/java/api/value/PlcValue.java

```diff
@@ -149,4 +149,10 @@ public interface PlcValue {
 
     Map<String, ? extends PlcValue> getStruct();
 
+    Map<String, PlcValue> getMetaData();
+
+    boolean hasMetaData(String key);
+
+    PlcValue getMetaData(String key);
```

Comment:
not sure if those are worth it if you expose a map anyway. This just bloats the API

Reply:
Guess I mainly added it, because we did it in the rest of the API.

Reply:
I actually removed the raw map access.